README.md

```diff
@@ -513,10 +515,28 @@ Raises `Patron::TimeoutError`
 ### rest-client
 
 ```ruby
-RestClient::Request.execute(method: :get, url: url, open_timeout: 1, timeout: 1)
+RestClient::Request.execute(method: :get, url: url, open_timeout: 1, read_timeout: 1)
+
+# shorthand to set open_timeout = read_timeout = 1
+RestClient::Request.execute(method: :get, url: url, timeout: 1)
 ```
 
-Raises `RestClient::RequestTimeout`
+Same options also work with `RestClient::Resource`.
+
+Raises:
+
+- `RestClient::Exceptions::OpenTimeout` on connect timeout
+- `RestClient::Exceptions::ReadTimeout` on read timeout
+
+both inherit from `RestClient::Exceptions::Timeout`.
+
+Defaults: inherited from net/http.
+
+**Note:** Based on net/http so read timeouts are retried automatically.
```

Comment:
This applies to many libraries listed in the guide. While accurate, seems repetitive to include it for each of them.